spec/lib/appsignal/js_exception_transaction_spec.rb

```diff
@@ -78,6 +78,46 @@
    end
   end
 
+  context "when sending just the name" do
+    let(:data) { {'name' => 'TypeError'} }
+
+    describe "#set_action" do
+      it "should not call `Appsignal::Extension.set_action`" do
+        expect( transaction.ext ).to_not receive(:set_action)
+
+        transaction.set_action
+      end
+    end
+
+    describe "#set_metadata" do
+     it "should not call `Appsignal::Extension.set_transaction_metadata`" do
+       expect( transaction.ext ).to_not receive(:set_metadata)
+
+       transaction.set_metadata
+     end
```

Comment:
Indenting is off, only 1 space instead of 2. Also for the specs below
Sorry, I spot these kinds of things